cockatrice/src/window_sets.cpp

```diff
@@ -128,11 +128,13 @@ WndSets::WndSets(QWidget *parent) : QMainWindow(parent)
     labNotes->setText(
         "<b>" + tr("Deck Editor") + ":</b> " +
         tr("Only cards in enabled sets will appear in the deck editor card list") + "<br><br>" + "<b>" +
-        tr("Card Art") + ":</b> " + tr("Image priority is decided in the following order") + "<ol><li>" + tr("The") +
-        "<a href='https://github.com/Cockatrice/Cockatrice/wiki/"
-        "Custom-Cards-%26-Sets#to-add-custom-art-for-cards-the-easiest-way-is-to-use-the-custom-folder'> " +
-        tr("CUSTOM Folder") + "</a></li><li>" + tr("Enabled Sets (Top to Bottom)") + "</li><li>" +
-        tr("Disabled Sets (Top to Bottom)") + "</li></ol>");
+        tr("Card Art") + ":</b> " + tr("Image priority is decided in the following order") + "<ol><li>" +
+        tr("The %1 CUSTOM Folder")
+            .arg("<a "
+                 "href='https://github.com/Cockatrice/Cockatrice/wiki/"
+                 "Custom-Cards-%26-Sets#to-add-custom-art-for-cards-the-easiest-way-is-to-use-the-custom-folder'> ") +
+        "</a></li><li>" + tr("Enabled Sets (Top to Bottom)") + "</li><li>" + tr("Disabled Sets (Top to Bottom)") +
```

Comment:
That could be very tricky to translate in all languages, and users probably won't understands what it's about.
I think we have to rephrase to avoid problems, because the former solution was also not ideal.

Reply:
The proposed solution is not perfect, but surely better than the previous one: at least you are not forced to put the article at the beginning of the sentence.

Reply:
What about simply removing the leading article and have it say `CUSTOM Folder`?

I would even remove the CUSTOM part from the translation because we name that in our code base (so it will never change or adjust to the users language), and only translate `folder`... :)

Reply:
I instead opted to put the entire block in a single tr() call, sometimes the simplest solutions are best.

Reply:
Not so sure what will happen with the html formatting...
Translators could mess up a lot here, and the single section is quite big!